src/app/organisations/deleteUserOrganisation.js

```diff
@@ -1,11 +1,31 @@
-const { deleteUserOrganisation } = require('./data/organisationsStorage');
+const { deleteUserOrganisation, getServiceAndSubServiceReqForOrgs, updateUserServSubServRequest } = require('./data/organisationsStorage');
 
 const deleteOrg = async (req, res) => {
-  const organisationId = req.params.id;
-  const userId = req.params.uid;
+  const correlationId = req.get('x-correlation-id');
+  const { id: orgId, uid: userId } = req.params;
```

Comment:
is User id being validated as not null or undefined?

Reply:
A guard clause has been introduced in the implementation for both the orgId and userId params and the tests have been updated to match.